build.sbt

```diff
@@ -29,8 +30,9 @@ lazy val root = (project in file("."))
       "org.typelevel"     %% "cats-effect"                % catsEffectVersion,
       "co.fs2"            %% "fs2-core"                   % fs2Version,
       "co.fs2"            %% "fs2-io"                     % fs2Version,
+      "com.chuusai"       %% "shapeless"                  % shapelessVersion,
```

Comment:
why do we need `shapless` ?

Reply:
That is a good question, **sbt-explicit-dependencies** says this project depends directly _(meaning there is a direct import somewhere)_ on **Shapeless**.
Probably there is a derivation somewhere.

Reply:
ok. weird. I know `scodec` uses it.